coverage.sh

```diff
@@ -5,21 +5,16 @@ while getopts "v:" arg; do
   esac
 done
 
-yarn buidler clean
+yarn hardhat clean
 rm -r ./.tempCoverageArtifacts
 
 if [ "$VERSION" == 'sol4' ]; then
   echo "Running sol4 coverage..."
-  yarn buidler coverage --config ./buidlerConfigSol4.js --testfiles "test/sol4/*.js" --solcoverjs ".solcover.js"
+  yarn hardhat coverage --testfiles "test/sol4/*.js" --solcoverjs ".solcover.js"
 elif [ "$VERSION" == 'sol5' ]; then
-  yarn buidler compile --config buidlerConfigSol4.js
-  cp -R ./artifacts/ ./.tempCoverageArtifacts
   echo "Running sol5 coverage..."
-  yarn buidler coverage --config ./buidlerConfigSol5.js --testfiles "test/sol5/*.js" --solcoverjs ".solcover.js"
+  yarn hardhat coverage --testfiles "test/sol5/*.js" --solcoverjs ".solcover.js"
 else
   echo "Running sol6 coverage..."
-  yarn buidler compile --config buidlerConfigSol5.js &&
-  yarn buidler compile --config buidlerConfigSol4.js
-  cp -R ./artifacts/ ./.tempCoverageArtifacts
-  yarn buidler coverage --config ./buidlerConfigSol6.js --testfiles "test/sol6/*.js" --solcoverjs ".solcover.js"
+  yarn hardhat coverage --testfiles "test/sol6/*.js" --solcoverjs ".solcover.js"
 fi
```

Comment:
does this create full reports for all sol versions?

Reply:
There's some issue with coverage, upgrading to latest version as it provides hardhat support